packages/ariakit/src/heading/__examples__/heading/style.css

```diff
@@ -0,0 +1,3 @@
+.heading1{
+    @apply text-primary-1
+}
```

Comment:
Since we don't know the heading levels statically, I'd say it's okay to style `h1`, `h1`, `h3`, `h4` etc., let's give them different sizes.